api_v3/lib/syndication/KalturaSyndicationFeedRenderer.php

```diff
@@ -143,6 +143,16 @@ public function __construct($feedId, $feedProcessingKey = null, $ks = null, $sta
 		
 		myPartnerUtils::resetPartnerFilter('entry');
 
+		if($this->shouldAddNextLink())
+		{
+			$this->addLinkForNextIteration = true;
+
+			if($this->syndicationFeed->entryFilter && ($this->syndicationFeed->entryFilter->statusEqual || $this->syndicationFeed->entryFilter->statusIn))
+			{
+				entryPeer::allowDeletedInCriteriaFilter();
```

Comment:
add a call to entryPeer::blockDeletedInCriteriaFilter when you no longer need this enabled otherwise this might screw up the next request in a multirequest scenario.